docs/user_guide.md

```diff
@@ -701,18 +672,7 @@ void test() {
   --export-source-suppress
```

Comment:
`export-source-suppress` works at the analysis phase (collects suppress comments) and does not get the values from the database. I think we should not remove this yet. At least not as part of this pull request.